listenbrainz_spark/recommendations/recommend.py

```diff
@@ -277,31 +281,78 @@ def create_messages(top_artist_rec_mbid_df, similar_artist_rec_mbid_df, active_u
 
         if user_rec.get(row.user_name) is None:
             user_rec[row.user_name] = {}
-            user_rec[row.user_name]['top_artist'] = [[row.mb_recording_mbid, row.rating]]
+            try:
+                user_rec[row.user_name]['top_artist'] = [
+                    UserRecommendationsRecord(**{
+                        "recording_mbid": row.mb_recording_mbid,
+                        "score": row.rating
+                    }).dict()
+                ]
+            except ValidationError:
+                current_app.logger.warning("""Invalid entry present in top artist recommendations for user: {user_name},
+                                           skipping""".format(user_name=row.user_name), exc_info=True)
+
             user_rec[row.user_name]['similar_artist'] = []
 
         else:
-            user_rec[row.user_name]['top_artist'].append([row.mb_recording_mbid, row.rating])
+            try:
+                user_rec[row.user_name]['top_artist'].append(
+                    UserRecommendationsRecord(**
```

Comment:
why are we expecting an error here? going from dict -> pydantic object -> dict seems redundant. maybe it makes sense to just keep a dict here

Reply:
Hmm. Can you merge and restart request consumer for now? I will update this later.  I guess you won't be available in the morning to restart request consumer, and I want to test stuff for a user whose recommend tracks page raise ISE.

Reply:
Done! Can you open a ticket (or even better, a PR) addressing the comment?